app/views/candidate_mailer/application_submitted_apply_again.text.erb

```diff
@@ -1,11 +1,23 @@
 Dear <%= @application_form.first_name %>,
 
-You’ve submitted an application for <%= @application_choice.course_option.course.name_and_code %> at <%= @application_choice.course_option.course.provider.name %>.
-
-Your application reference is <%= @application_form.support_reference %>.
+You have submitted an application for <%= @application_choice.course_option.course.name_and_code %> at <%= @application_choice.course_option.course.provider.name %>.
 
 Sign in to your account to check the progress of your application:
 
 <%= candidate_magic_link(@application_form.candidate) %>
 
-If your training provider decides to progress your application, they’ll contact you to organise an interview.
+<% if @application_form.application_choices.count > 1 %>
+
+  If your training providers decide to progress your application, they’ll contact you to organise an interview.
+
+<% else %>
+
+  If your training provider decides to progress your application, they’ll contact you to organise an interview.
+
+<% end %>
```

Comment:
We shouldn't need this condition, as you can only have 1 choice in Apply again?

Reply:
good point! (although, we might be changing business rules so that you can apply for more than one in the not too distant future)

Reply:
ooh thats a good point, that slipped my mind. Shall we remove it or leave it in for future proofing? 🤔

Reply:
Discussed offline - the decission is to remove the conditional on line 9 as currently you can only have one choice in Apply again

Reply:
Changes made